controlsfx/src/main/java/impl/org/controlsfx/tableview2/TableRow2Skin.java

```diff
@@ -245,174 +246,170 @@ private void requestCellUpdate() {
                 if (hbarValue + fixedColumnWidth > x) {
                     increaseFixedWidth = true;
                     tableCellX = Math.abs(hbarValue - x + fixedColumnWidth);
-//                	 tableCell.toFront();
+//                	tableCell.toFront();
                     fixedColumnWidth += width;
-//                    isVisible = true; // If in fixedColumn, it's obviously visible
+//                  isVisible = true; // If in fixedColumn, it's obviously visible
                     fixedCells.add(tableCell);
                 }
             }
-            
-            if (isVisible) {
```

Comment:
Why is this check removed?

Reply:
Because of line 213 the visible variable was alway true at this point.